src/Glpi/Database/AbstractDatabase.php

```diff
@@ -732,6 +767,47 @@ public function fieldExists(string $table, string $field, bool $usecache = true)
         return false;
     }
 
+    /**
+     * Check if an index exists
+     *
+     * @since 10.0
+     *
+     * @param string       $table    Table name for the field we're looking for
+     * @param string|array $field    Field(s) name(s)
```

Comment:
I am not a great fan of mixed type parameters. I would prefer to accept only an array, to be able to make PHP throw a fatal error on bad usage (this kind of function is used in plugin installers).

Reply:
I've initially kept the mixed, but indeed, as this is a new method, we can require an array only here.

Reply:
Thinking about that, thins seems finally a bad idea. Expet from automatic usage from Migration, the common usage is to use a string; this seems simplier to keep a string.